### DISCUSSION TITLE
WFMZ Weather Channel

### DISCUSSION THREAD

**Discussion by arkina1234-undenified** [iptv-org/community#370] [Show and tell]
### Channel ID

WFMZDT2.us

### Stream URL

https://dai2-playlistserver.aws.syncbak.com/cpl/17869668/dai2v5/1.0/7b2264657669636554797065223a382c226361706162696c6974696573223a7b7d7d/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjIwMTYwNDAxIn0.eyJkZXZpY2VUeXBlIjo4LCJzaWQiOiI5MSIsImNpZCI6MTY1MjUsIm9vbSI6dHJ1ZSwiYXBzdCI6bnVsbCwic3NpZCI6Ijg3NDZhOTdiMmFkNjQyYWU5YjE1ZmY4Mjc0OTZhY2E4IiwiaXAiOiI1Mi43Mi4xNTUuMjMyIiwidWlwIjoiMjE3LjEzOC4yMDYuODUiLCJkbWEiOjUwMSwic24iOiJXZWIgMTQ4YTk3MjAwZiAiLCJ1aWQiOm51bGwsImNzYiI6ZmFsc2UsIm1pZCI6IjE3ODY5NjY4IiwibWQ1IjoiZDMwNjZkMjJjZjNmZWQ2OTE1NzQ1MWEyYTQ5NTBhZWUiLCJpYXQiOjE2NzE1NTA1MzcsImV4cCI6MTY3MTU1MDYyNywiaXNzIjoiV0xTIiwic3ViIjoiMjYzOGQ1ZDMtNTc0Mi00YzNkLWE4MzYtN2U1ZDYxM2E5MDIwIn0.BgbNsOvTJPu1VDRpexh3fZxoRsoLinX3kJKMBAPBWvU

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by arkina1234-undenified** [comment]
Tokened

**Comment by freearhey** [comment]
https://dai2-playlistserver.aws.syncbak.com/cpl/17869668/dai2v5/1.0/7b2264657669636554797065223a382c226361706162696c6974696573223a7b7d7d/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjIwMTYwNDAxIn0.eyJkZXZpY2VUeXBlIjo4LCJzaWQiOiI5MSIsImNpZCI6MTY1MjUsIm9vbSI6dHJ1ZSwiYXBzdCI6bnVsbCwic3NpZCI6Ijg3NDZhOTdiMmFkNjQyYWU5YjE1ZmY4Mjc0OTZhY2E4IiwiaXAiOiI1Mi43Mi4xNTUuMjMyIiwidWlwIjoiMjE3LjEzOC4yMDYuODUiLCJkbWEiOjUwMSwic24iOiJXZWIgMTQ4YTk3MjAwZiAiLCJ1aWQiOm51bGwsImNzYiI6ZmFsc2UsIm1pZCI6IjE3ODY5NjY4IiwibWQ1IjoiZDMwNjZkMjJjZjNmZWQ2OTE1NzQ1MWEyYTQ5NTBhZWUiLCJpYXQiOjE2NzE1NTA1MzcsImV4cCI6MTY3MTU1MDYyNywiaXNzIjoiV0xTIiwic3ViIjoiMjYzOGQ1ZDMtNTc0Mi00YzNkLWE4MzYtN2U1ZDYxM2E5MDIwIn0.BgbNsOvTJPu1VDRpexh3fZxoRsoLinX3kJKMBAPBWvU (offline)
<img width="842" alt="image" src="https://user-images.githubusercontent.com/7253922/232248407-a15a7907-954f-4b19-9ab3-c703bc76c67d.png">
